integration-tests/test_integration/client-integration.test.ts

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
[NIT] I don't like seeing both kebab case and snake case in the same path.

Reply:
Yep, agreed, will clean this up before putting the PR in initial review.

Reply:
We earlier decided on lower camelCase for all the file names. And we named all the source files in one PR.